### ISSUE TITLE
Driver constructor with default argument `""` triggers false positive

### ISSUE THREAD

**Issue by nbenn** [r-dbi/DBItest#293]
The check performed at

https://github.com/r-dbi/DBItest/blob/8c67927b6f98a40ff475410f3400265d9d5e122b/R/expectations.R#L18

confuses a default value of `""` with having no default at all.

**Comment by krlmlr** [comment]
Yeah, it's terrible. What does the correct implementation look like?
